### PR TITLE
fix(error handling): display error page if backend not available

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Display Error page if backend not available

## Why

When backend isn't available, the app continuously sends thousands of requests

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/632

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

**Review by evegufy — CHANGES_REQUESTED** [review]
Please wait for approval from @jjeroch

**Review by oyo — APPROVED** [review]

**Comment by nidhigarg-bmw** [comment]
@evegufy @jjeroch The Error page will look like this
image
And this page will only work if backend isnt available. And it wont affect other error handlings and functionalities which are working before.

**Review by evegufy — APPROVED** [review]

**Pull request merged**
